src/sql/workbench/contrib/scripting/browser/scripting.contribution.ts

```diff
@@ -164,6 +164,7 @@ MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
 				TreeNodeContextKey.NodeType.isEqualTo(NodeType.TableValuedFunction)),
+			TreeNodeContextKey.NodeType.isEqualTo(NodeType.Function)
```

Comment:
does this need to be scoped to a provider like all the other rules in the expression?